### ISSUE TITLE
Klaviyo Unsubscribe Pages Blocked by List

### ISSUE THREAD

**Issue by chas-klaviyo** [lassekongo83/Frellwits-filter-lists#153]
### URL(s) where the issue occurs

`Link: [https://manage.kmail-lists.com/subscriptions/unsubscribe?a=QYsFmX&c=01GPH26YVDB2GCWD8C1KQS9H24&k=4e216af999861a7ca47da8825588d418&m=01GPJVG20A78X9JS6DQQFM5NC7&r=SRA78Vv]`

### Describe the issue

Hello maintainers, I'm an engineer at [Klaviyo](https://www.klaviyo.com/) and we recently noticed that our unsubscribe pages do not function correctly when used in conjunction with this ad block list. 

Our unsubscribe pages leverage some javascript loaded from `https://static.klaviyo.com/onsite-consent-pages/js/onsiteConsentPages.js` which is blocked by this list, as this list has [this filter](https://github.com/lassekongo83/Frellwits-filter-lists/blob/master/Frellwits-Swedish-Filter.txt#L2086): `||klaviyo.com^$third-party`.

This filter should instead be updated to `||static-tracking.klaviyo.com^$third-party` so that it only blocks the areas where any tracking occurs. We see this from time to time on different privacy lists, which is why we moved all tracking JS and http requests onto the specific static-tracking sub-domain. For example adding a.klaviyo and static.klaviyo to duck duck go's allow list: https://github.com/duckduckgo/privacy-configuration/issues/551

### Versions

- Browser/version: Google Chrome Version 109.0.5414.87 (Official Build) (x86_64)
- uBlock Origin version: 1.46.0

### Settings

Enabled this list in uBO

**Comment by lassekongo83** [comment]
This filter also blocks annoyances, such as modal popups related to e-mail marketing, so for now I'm just going to put an exception on the unsubscribe site.

**Comment by chas-klaviyo** [comment]
@lassekongo83 Hey Mattias, thanks for the quick response here! There's one more exception we'd like to request, as it's possible for our customers to set up unsubscribe pages on custom subdomains as well, and those are served from the myklpages.com domain. Would it be possible to update the entry like you did above to 

```
||klaviyo.com^$third-party,domain=~kmail-lists.com,domain=~myklpages.com
```

Also happy to open another issue if that's easier.
